README.md

```diff
@@ -127,34 +127,10 @@ See the [CONTRIBUTING] document. Thank you, [contributors]!
 [CONTRIBUTING]: CONTRIBUTING.md
 [contributors]: https://github.com/thoughtbot/Liftoff/graphs/contributors
 
-Need Help?
-----------
-
-We offer 1-on-1 coaching. We can help you with functional programming in Swift,
-get started writing unit tests, and convert from Objective-C to Swift.
-[Get in touch].
-
-[Get in touch]: http://coaching.thoughtbot.com/ios/?utm_source=github
-
-License
--------
-
-Liftoff is Copyright (c) 2015 thoughtbot, inc. It is free software, and may be
-redistributed under the terms specified in the [LICENSE] file.
```

Comment:
I don't think there _is_ a copyright? I don't know how that works for community owned projects. The license hasn't changed, but this mention seemed unnecessary.